docs/usage/configuration-options.md

```diff
@@ -1492,6 +1492,24 @@ A use case for the latter is if you are a Renovate bot admin and wish to provide
 If `false` (default), it means that defining `config.npmrc` will result in any `.npmrc` file in the repo being overridden and its values ignored.
 If configured to `true`, it means that any `.npmrc` file in the repo will have `config.npmrc` prepended to it before running `npm`.
 
+## osvVulnerabilityAlerts
+
+Renovate integrates with [OSV](https://osv.dev/), an open-source vulnerability database, to check if extracted dependencies include known vulnerabilities.
+Set this option to `true` to receive pull requests with vulnerability fixes once they are available.
```

Comment:
```suggestion
Set `osvVulnerabilityAlerts` to `true` to get pull requests with vulnerability fixes.
```

Do we need to mention "once (security fixes) are available"? If there's no security fix, there's nothing for Renovate to put in the PR. At least that is clear to me, but maybe not to others?

Reply:
Vulnerabilities are very often published when there's no fix available yet. "once they are available" was meant to prevent user questions on "why didn't renovate create a PR that fixes vulnerability ABC". Besides that, the idea was to hint that renovate *continuously* checks for fixes.

Reply:
Thank you for the background information!

```suggestion
Set `osvVulnerabilityAlerts` to `true` to get pull requests with vulnerability fixes (once they are available).
```

How about this then? 😉